scripts/ChangesSinceLastRelease.txt

```diff
@@ -236,3 +236,296 @@ Windows.Win32.System.Threading.Apis.CreateProcessAsUserW : dwCreationFlags...UIn
+Windows.Win32.System.ApplicationInstallationAndServicing.Apis.MsiGetPatchFileListA : pphFileRecords : [Optional,Out] => [Out]
+Windows.Win32.System.ApplicationInstallationAndServicing.Apis.MsiGetPatchFileListW : pphFileRecords : [Optional,Out] => [Out]
+Windows.Win32.System.Com.Apis.CoTaskMemFree : pv : [Optional] => [In,Optional]
+Windows.Win32.System.Com.Apis.CoTaskMemRealloc : pv : [Optional] => [In,Optional]
+Windows.Win32.System.Com.CallObj.ICallFrame.GetNames : pwszInterface : [Optional,Out] => [Out]
+Windows.Win32.System.Com.CallObj.ICallFrame.GetNames : pwszMethod : [Optional,Out] => [Out]
+Windows.Win32.System.Com.CallObj.ICallIndirect.GetIID : pwszInterface : [Optional,Out] => [Out]
+Windows.Win32.System.Com.CallObj.ICallIndirect.GetMethodInfo : pwszMethod : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IClientSecurity.QueryBlanket : pAuthInfo : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IRpcChannelBuffer.GetDestCtx : ppvDestContext : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IServerSecurity.QueryBlanket : pPrivs : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetFragment : ppwzFragment : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetHost : ppwzHost : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetIUri : ppIUri : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetPassword : ppwzPassword : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetPath : ppwzPath : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetQuery : ppwzQuery : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetSchemeName : ppwzSchemeName : [Optional,Out] => [Out]
+Windows.Win32.System.Com.IUriBuilder.GetUserName : ppwzUserName : [Optional,Out] => [Out]
+Windows.Win32.System.Com.Urlmon.IInternetHostSecurityManager.QueryCustomPolicy : ppPolicy : [NativeArrayInfo(CountParamIndex=2),Optional,Out] => [NativeArrayInfo(CountParamIndex=2),Out]
+Windows.Win32.System.Diagnostics.Debug.Apis.MiniDumpReadDumpStream : Dir : [Optional,Out] => [Out]
+Windows.Win32.System.Diagnostics.Debug.Apis.MiniDumpReadDumpStream : StreamPointer : [Optional,Out] => [Out]
+Windows.Win32.System.Diagnostics.Debug.IDynamicConceptProviderConcept.GetConcept : conceptInterface : [ComOutPtr,Optional,Out] => [ComOutPtr,Out]
+Windows.Win32.System.Diagnostics.Debug.IModelObject.GetContext : context : [ComOutPtr,Optional,Out] => [ComOutPtr,Out]
+Windows.Win32.System.Diagnostics.Debug.IModelObject.GetParentModel : contextObject : [ComOutPtr,Optional,Out] => [ComOutPtr,Out]
+Windows.Win32.System.Diagnostics.Etw.Apis.TdhCreatePayloadFilter : PayloadFilter : [Optional,Out] => [Out]
+Windows.Win32.System.EventLog.Apis.EvtSubscribe : Callback : [Optional] => [In]
+Windows.Win32.System.EventLog.Apis.EvtSubscribe : Context : [Optional] => [In,Out]
+Windows.Win32.System.EventLog.EVT_SUBSCRIBE_CALLBACK.Invoke : UserContext : [Optional] => [In,Out]
```

Comment:
Callback is marked as `_Maybenull_` in the headers. This can be set to `null` or point to a blob somewhere. This seems to qualify for the `[Optional]` attribute, right?

Reply:
We can check for _maybenull_ by itself and add optional for cases like this.